lldb/source/ValueObject/DILEval.cpp

```diff
@@ -272,4 +272,204 @@ Interpreter::Visit(const UnaryOpNode *node) {
       m_expr, "invalid ast: unexpected binary operator", node->GetLocation());
 }
 
+lldb::ValueObjectSP
+Interpreter::EvaluateMemberOf(lldb::ValueObjectSP value,
+                              const std::vector<uint32_t> &path,
+                              bool use_synthetic, bool is_dynamic) {
```

Comment:
`use_dynamic` was passed in the the parser as a `lldb::DynamicValueType`. Why did it turn into a `bool` here?

Reply:
Removed this function so comment doesn't matter now.